src/meta/src/barrier/mod.rs

```diff
@@ -309,7 +309,7 @@ impl CheckpointControl {
             );
         }
 
-        debug!(
+        tracing::trace!(
```

Comment:
Note that logs with `trace` level will be optimized out in the release build.

Reply:
Do you think we may need it in prod?

Reply:
Not sure 🤣